media-sound/pamix/pamix-1.6-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == 9999 ]] ; then
```

Comment:
It'd be nicer to merge this with the existing block but this isn't your package, so I get the reluctance to meddle too much with style

Reply:
agreed, also `if [[ ${PV} == *9999 ]] ; then` please

Reply:
yes, that's how I would do it, but rather leave it up to @Polynomial-C

Reply:
I have no strong opinion. Leave it like this or change it but please don't break it :)